charts/keycloak-operator/templates/operator/clusterrole.yaml

```
@@ -0,0 +1,43 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: keycloakcontroller-cluster-role
+  labels:
+    {{- include "keycloak-operator.operator.labels" . | nindent 4 }}
+rules:
+  - apiGroups:
+      - k8s.keycloak.org
+    resources:
+      - keycloaks
+      - keycloaks/status
+      - keycloaks/finalizers
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - delete
+      - patch
+      - update
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: keycloakrealmimportcontroller-cluster-role
+  labels:
+    {{- include "keycloak-operator.operator.labels" . | nindent 4 }}
+rules:
+  - apiGroups:
+      - k8s.keycloak.org
+    resources:
+      - keycloakrealmimports
+      - keycloakrealmimports/status
+      - keycloakrealmimports/finalizers
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - delete
+      - patch
+      - update
```

I think giving the operator all permissions would be acceptable here:

```suggestion
      - '*'
```

---

same here, it's based on [this](https://github.com/keycloak/keycloak-k8s-resources/blob/02a2913e2ad9e3454844bd8f6bb0ebfb03765c0e/kubernetes/kubernetes.yml#L157-L175)